conda/plugins/types.py

```diff
@@ -62,3 +65,17 @@ class CondaPreCommand(NamedTuple):
     name: str
     action: Callable
     run_for: set[str]
+
+
+class CondaPostCommand(NamedTuple):
+    """
+    Allows a plugin hook to execute after an invoked conda command is run.
+
+    :param name: Post-command name (e.g., ``custom_plugin_post_commands``).
+    :param action: Callable which contains the code to be run.
+    :param run_for: Represents the command(s) this will be run on (e.g. install or create).
+    """
+
+    name: str
+    action: Callable
+    run_for: set[str]
```

Comment:
Do you think it would be useful to add a feature to optionally receive the result of the command?

Reply:
What would the result be? I think most commands either return `None` or throw an exception if there was an error. In the latter case, the `post_command` would never be invoked. Should that behavior be different? (call the `post_command` regardless of whether the command succeeded or not?